java/ql/src/Security/CWE/CWE-502/UnsafeDeserialization.qhelp

```diff
@@ -28,11 +28,64 @@ for example JSON or XML.  However, these formats should not be deserialized
 into complex objects because this provides further opportunities for attack.
 For example, XML-based deserialization attacks
 are possible through libraries such as XStream and XmlDecoder.
-
+</p>
+<p>
 Alternatively, a tightly controlled whitelist can limit the vulnerability of code, but be aware
 of the existence of so-called Bypass Gadgets, which can circumvent such
 protection measures.
 </p>
+<p>
+Fixes by framework:
```

Comment:
```suggestion
Recommendations specific to particular frameworks supported by this query:
```

(So, should we make sure we say something about every framework mentioned in the `<overview>` section, even if it's just to note that we don't look for any particular configuration and just alert on all uses with potentially-tainted input?)